src/core/bsearch.rkt

```diff
@@ -76,16 +76,17 @@
        (with-handlers ([exn:fail:user:herbie? (const 'fail)])
          (pred p3)))
 
-     (cond
-       [(eq? cmp 'fail)
-        (timeline-push! 'stop "predicate-failed" 1)
-        (values p1 p2)]
-       [(negative? cmp) (binary-search-floats pred p3 p2 repr)]
-       [(positive? cmp) (binary-search-floats pred p1 p3 repr)]
-       ;; cmp = 0 usually means sampling failed, so we give up
-       [else
-        (timeline-push! 'stop "predicate-same" 1)
-        (values p1 p2)])]))
+    (cond
+     [(eq? cmp 'fail)
+      (timeline-push! 'stop "predicate-failed" 1)
+      (values p1 p2)]
+     [(negative? cmp) (binary-search-floats pred p3 p2 repr)]
+     [(positive? cmp) (binary-search-floats pred p1 p3 repr)]
+     ;; cmp = 0 usually means sampling failed, so we give up
+     [else
+      (timeline-push! 'stop "predicate-same" 1)
+      (values p1 p2)])])
+      )
```

Comment:
What changed, besides the final parenthesis?

Reply:
Nothing it was an accident when I was trying to merge main into this branch.